basic-algorithms-&-datastructures/index030.js

```diff
@@ -0,0 +1,15 @@
+const newsFeed = {
+    page: "front_page",
+    articles: [{},{},{},{},{},{}]}
+
+/*
```

Comment:
This step could probably be broken up into two smaller steps. The first could read:

/*
Nested objects are objects within other objects. In the first nested empty object within the articles array, add the following properties: `headline`, `image`, `date_published`, `date_modified`, `author`, `likes`, `dislikes`, `shares`, `comments`, `sections`, and `url`.

Set `image` to an empty array and both `author` and `section` to empty objects. Assign all other properties the value `null`.
*/

And the next step could be:

/*
In the `author` object, add the properties `username` and `name` and set both to the value `null`.

Also in `section`, add the properties `name` and `subscribers` and set them both to `null`.
*/